src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -557,13 +554,3 @@ export const UpdateApplicationButton = ({
     )
 }
 
-export const ErrorScreenWithInfo = ({ info }: ErrorScreenWithInfoProps) => {
```

Comment:
Was it unused?

Reply:
I replaced this function already with GenericEmpty state where it was used so there is no use of this function.